# Define the goal
uint64 id_ui                # schema id
uint64 finalTargetRFID_ui   # final RFID
---
# Define the result
int8 errorCode_i            # error specification
---
# Define a feedback message
# no feedback required